src/governance.cpp

```diff
@@ -281,6 +281,9 @@ void CGovernanceManager::ProcessMessage(CNode* pfrom, std::string& strCommand, C
     // A NEW GOVERNANCE OBJECT VOTE HAS ARRIVED
     else if (strCommand == NetMsgType::MNGOVERNANCEOBJECTVOTE)
     {
+        // Ignore such messages until masternode list is synced
+        if(!masternodeSync.IsMasternodeListSynced()) return;
+
```

Comment:
Is this important ?  

It's not clear to me what the effect of processing govobj's when the MN list is not synced is.